### PR TITLE
Fix pylint's super-with-arguments

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: h/models/group.py [hunk 1]
```diff
@@ -152,9 +152,6 @@ def groupid(self, value):
     )
     organization = sa.orm.relationship("Organization")
 
-    def __init__(self, **kwargs):
```
Comment (marcospri) [inline h/models/group.py:155]: I don't think this was doing anything but I might be wrong.

File: h/services/user_unique.py [hunk 2]
```diff
@@ -5,9 +5,6 @@
 class DuplicateUserError(Exception):
     """Indicates that data violates user uniqueness constraints"""
 
-    def __init__(self, message):
```
Comment (marcospri) [inline h/services/user_unique.py:8]: Same here, I think this can be removed.

File: h/models/group.py (hunk 1 above)
Comment (jon-betts) [reply h/models/group.py:155]: Looks like nonsense to me!

**Review by jon-betts — APPROVED** [review]
This one is awesome

**Pull request merged**